core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/service/ClusterProxyService.java

```diff
@@ -146,35 +148,37 @@ private ConfigRegistrationRequest createProxyConfigReRegisterRequest(Stack stack
         return requestBuilder.build();
     }
 
+    // For CCM disabled and CCMv1 only.
```

Comment:
Next time please rename the function instead of the comment

Reply:
Thanks, fixed.